README.md

```diff
@@ -3,29 +3,76 @@
 [![Join the chat at https://gitter.im/Nithmr/ProxyHelper](https://badges.gitter.im/Nithmr/ProxyHelper.svg)](https://gitter.im/Nithmr/ProxyHelper?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 This repository will contain the code for a **cool** application that would remove the need to use FoxyProxy and likes.
 
+ProxyHelper is a set of utilities that you can **install and forget and it will just work!** It will automate all your proxy and tor configuration needs on linux. Meaning, you won't need to configure proxy for firefox, apt, git, etc manually, ever again.
 
-Pings multiple proxies in parallel and sets the fastest one for you!!
+Whether you are in AGH or CC or KBH, or your proxy is ```172.16.12.2``` or ```172.16.20.2``` or ```anything else```, all of the work is done in background and proxy is set for you as soon as you connect to the network, magically (and automatically).
+Another cool thing about ProxyHelper is that it always sets the *Fastest* proxy for you. This is especially relevant if you use networks in CC or EED and you don't know which proxy will be faster -```172.16.24.2``` or ```172.16.24.3```; leave it to ProxyHelper to test both proxies and set the faster proxy automatically.
+  
+Also, did I mention with ProxyHelper you can use internet with *tor* even after 2 A.M limit?(potentially to the next day!).   
```

Comment:
We definitely would like to delete this line. This will be displayed on publicly and will surely catch the eye of some in authority. So we cannot boast about using the internet after the set limit.